packages/frames.js/src/core/constants.ts

```diff
@@ -2,3 +2,5 @@
  * This meta tag is used to get flattened frames JSON representation that is easier to work with than parsing meta tags from HTML
  */
 export const FRAMES_META_TAGS_HEADER = "application/frames.js+metatags";
+
+export const FRAMES_IMAGES_PARAM_KEY = "__framesjs_jsx";
```

Comment:
isn't this a breaking change? Previously the key was `jsx`, it could break cached URIs right?